torch/csrc/jit/passes/onnx/list_model_parameters.cpp

```diff
@@ -143,6 +143,8 @@ std::vector<IValue> getParamAttributes(
           auto attrVal = tryInsertConstant(*graph, attr);
           n->output()->replaceAllUsesWith(*attrVal);
           n->destroy();
+          // Don't scan blocks below.
+          continue;
```

Comment:
Not at all sure, what this code does, but shouldn't we save this to say `nodes_to_destroy` list and iterate over it afterwards? (I.e. why there shouldn't be more nodes like that?)

Reply:
The continue just picks up with the next node.  If we're destroying this node we don't really care about scanning its sub-blocks, right?